presentations/en/2019-11-05-CIOEC-Mandate.md

```diff
@@ -93,64 +97,69 @@ By using the approaches and technologies we champion, we are able to better resp
 
 --
 
-Work in the open to avoid re-work
+### 1. Work in the open to avoid re-work
 
 - Use a GIT-based version control system for all software
 - Expose work in progress to other teams
 
 Note:
-ALL teams (developers, security, testing, operations) must use git.
-ALL Sharepoint sites must allow any ESDC employees read access by default - only lock down what is absouletly necessary
-Expose work in progress to improve team communication and share what you do with others to reduce administrative and organizational overhead
-Leverage [Values and Ethics Code for the Public Sector](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=25049) as foundational trusts with employees
-Encourages collaboration and fosters a culture of flexibility, mobility and well-being
-Explore open source tools for quick wins and the reduction of re-work
-Be open to contributions and constructive feedback from the get go
+
+- ALL teams (developers, security, testing, operations) must use git.
+- ALL Sharepoint sites must allow any ESDC employees read access by default -only lock down what is absolutely necessary
+- Expose work in progress to improve team communication and share what you do with others to reduce administrative and organizational overhead
+- Leverage [Values and Ethics Code for the Public Sector](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=25049) as foundational trusts with employees
+- Encourages collaboration and fosters a culture of flexibility, mobility and well-being
+- Explore open source tools for quick wins and the reduction of re-work
+- Be open to contributions and constructive feedback from the get go
 
 --
 
-Core competencies (continous learning)
+### 2. Core competencies (continous learning)
 
-- All IT teams must use at least 20% of their time to learn, to automate manual processes, or improve the way they work.
+- All IT teams must use at least 20% of their time to learn, to automate manual
+processes, or improve the way they work.
 
 Note:
-IT teams must dedicate time to learn/try new technologies, attend events/conferences and stay up to date on trends and technologies related to their work
-IT teams must dedicate time for automation of testing, deployments and other manual processes
-Empower multidisciplinary teams and allow them to work on new ideas in pursuit of business goals that solve important problems
+
+- IT teams must dedicate time to learn/try new technologies, attend
+- events/conferences and stay up to date on trends and technologies related to their work
+- IT teams must dedicate time for automation of testing, deployments and other manual processes
+- Empower multidisciplinary teams and allow them to work on new ideas in pursuit of business goals that solve important problems
 
 --
 
-Modern tools
+### 3. Modern tools
 
 - Architecture must default to open standards and open source software
 - Architecture must enable deploy-anytime capabilities and promote relentless use automation
```

Comment:
Includes API first, Cloud Native solutions? Something to those lines?

Reply:
@gcharest we could. the
https://sara-sabr.github.io/ITStrategy/strategies-actions.html
has a lot more listed though. 
I think the idea of this section is to empower employees with modern tools to work with. Instead of applying architectural principles?